.gitignore

```diff
@@ -28,3 +28,4 @@ _testmain.go
 
 # Coverage reports
 *.coverprofile
+.idea
```

Comment:
This is ok, but I opted out of that game permanently by creating a `~/.gitignore_global` containing:
```
*~
.DS_Store
.idea
*.iml
\#*#
*.hsp
*.sav
*.scpt
```
I don't see why individual projects should necessarily support IntelliJ but not other IDEs. Please decide for yourself - this is just a suggestion.